tutoraspects/templates/aspects/build/aspects-superset/Dockerfile

```diff
@@ -3,37 +3,30 @@
 # https://github.com/apache/superset/releases
 # https://github.com/apache/superset/blob/master/Dockerfile
 # https://superset.apache.org/docs/databases/installing-database-drivers
-FROM apache/superset:4.0.2
+FROM apache/superset:4.1.0rc2
```

Comment:
Just tagging this so no one merges, but we shouldn't take this change until at least 4.1.0 is released. It looks like there will be at least one more RC cycle until that happens, so we may want to make this a draft PR for now.